src/probnum/problems/zoo/quad/_quadproblems_uniform.py

```diff
@@ -0,0 +1,802 @@
+import itertools
+
+import numpy as np
+from scipy.stats import norm
+
+from probnum.typing import FloatArgType, IntArgType
+
+
```

Comment:
```suggestion
__all__ = [
    "genz_continuous",
    "genz_cornerpeak",
    "genz_discontinuous",
    "genz_gaussian",
    "genz_oscillatory",
    "genz_productpeak",
    "bratley1992",
    "roos_arnold",
    "g_function",
    "morokoff_caflisch_1",
    "morokoff_caflisch_2",
]
```

These are also the names I would suggest for the problem constructors